vm/systemSmartContracts/validator.go

```diff
@@ -1124,9 +1127,35 @@ func (v *validatorSC) unStakeNodesFromStakingSC(blsKeys [][]byte, registrationDa
 		}
 
 		numSuccess++
+
+		stakedData, err := v.getStakedData(blsKey)
+		if err != nil {
+			continue
+		}
+		if stakedData.UnStakedNonce > currentNonce {
+			numSuccessFromWaiting++
+		}
+	}
+
+	numSuccessFromActive := numSuccess - numSuccessFromWaiting
+	return numSuccessFromActive, numSuccessFromWaiting
+}
+
+func (v *validatorSC) computeProcessUnStakeFromNodes(
```

Comment:
maybe rename as computeProcess... doesn't tell much :D

Reply:
I do not have a better idea

Reply:
renamed